src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncCLI.scala

```diff
@@ -76,44 +111,95 @@ object HdfsRsyncCLI {
 
         opt[Unit]('v', "verbose")
             .optional()
-            .action((_, c) => c.copy(logLevel = Level.DEBUG))
-            .text("Add verbosity to logging (DEBUG)")
+            .action((_, c) => c.copy(applicationLogLevel = Level.DEBUG))
+            .text("Add verbosity to application logging (DEBUG)")
 
         opt[Unit]('q', "quiet")
             .optional()
-            .action((_, c) => c.copy(logLevel = Level.WARN))
-            .text("Remove verbosity from logging (WARN)")
+            .action((_, c) => c.copy(applicationLogLevel = Level.WARN))
+            .text("Remove verbosity from application logging (WARN)")
+
+        opt[Unit]("all-logs-info")
+            .optional()
+            .action((_, c) => c.copy(rootLogLevel = Level.INFO))
+            .text("Set root logger level to INFO (default: ERROR)")
+
+        opt[Unit]("all-logs-debug")
```

Comment:
Perhaps it would be best to just use an option that takes a log level?  --log-level=debug and/or --log-level-all=debug ?

Reply:
Doing that